### DISCUSSION TITLE
Uploaded proof of academic status - no identication upload worked...?

### DISCUSSION THREAD

**Discussion by rjp23** [community/community#126276] [GitHub Education] [answered]
### Hi everyone,

I've uloaded proof of my academic status but there's no confirmation from the upload form that the upload worked. When I revisit the page, it just asks me to upload again.

Should I be seeing a confirmation page?

It doesn't seem like it's possible to talk to a person any more via the Contact Page.

Thanks!

**Comment by davevad93** [comment]
Hi @rjp23 ,

If the option to upload the documentation appears again when you revisit the application page, it could be for two reasons:

1. Your request hasn't been submitted correctly.
2. Your request has been submitted but hasn't been accepted for some reason, in that case GitHub usually notifies you by email that the documentation provided is not enough to prove your academic status or that some essential info is missing and on the application page to the right, you will see something like this below.

  ![1](https://github.com/community/community/assets/94308234/f318f96e-24a4-4c1b-964e-993cdf0f493c)

When the application has been submitted correctly there's no confirmation page, in that same page it appears a message that your request has been submitted and awating for approval (you will also receive a notification email). If your application has been approved you will see a notification card like the one below stating: `Congrats! Your request for @rjp23 was approved and your benefits will be available soon.`. 

![2](https://github.com/community/community/assets/94308234/820ee188-3d9c-4ae4-ad20-dcba442d6501)

When your benefits are fully available the notification changes to `Congrats! Your request for @rjp23 was approved and your benefits are now available.`

![3](https://github.com/community/community/assets/94308234/74fe967b-5a77-4d0c-9bb6-918b95734128)

Maybe it's a problem with your browser cookies, try with a different one.

**Comment by rjp23** [comment, marked as answer]
Thanks

The solution seems ridiculous but.... I used the webcam photo option instead and took of a photo of the document open on my phone screen and that worked and was accepted immediately.

It seems that the "upload" button the form is just broken and it doesn't trigger the processing of the application.

**Reply by alzinx** [reply]
Thank you for your feedback, had the same issue, I tried on 4 different browsers and 3 different devices and was waiting for more than two weeks.
I also used the webcam today after seeing your post and it also solved the problem for me !

**Reply by Yatharth-Nanda** [reply]
> Thanks
> 
> The solution seems ridiculous but.... I used the webcam photo option instead and took of a photo of the document open on my phone screen and that worked and was accepted immediately.
> 
> It seems that the "upload" button the form is just broken and it doesn't trigger the processing of the application.

Sorry how do i fix this again , im supposed to open the page on my laptop and then submit it ?

**Reply by ni-saantt** [reply]
I WAS STUCK IN THIS PROCESS FROM 3 DAYS ,THE SOUTION SOUNDS REALLY READICIOLUS BUT YA IT WORS , I WAS REALLY NOT EXPECTING THAT THIS WILL WORK

**Comment by adma77ya** [comment]
Now theres no upload button, we HAVE to use the take picture button and use the webcam....I tried this for the first time and was rejected becuase I hadnt filled my profile and billing info...Then it asked me to complete them and then log in back again after which the application would be accepted but whenever i click a pic and press 'process application' it just refreshes the page without submitting...I tried this so many times...fruatrated af

**Reply by HabiburRahaman07** [reply]
I'm facing the same problem and don't know how to solve it

**Reply by adma77ya** [reply]
try this ig https://github.com/orgs/community/discussions/129399#discussioncomment-9853042

**Comment by iammrk** [comment]
If github does not want to give Education discounts it is fine, it should not waste time of students by providing an option that does not work.
